source/api/cypress-api/custom-commands.md

```diff
@@ -186,8 +186,8 @@ By setting the `{ prevSubject: true }`, our new `.console()` command will requir
 Invoking it like this would error:
 
 ```javascript
-cy.console() // detailed error about how you can't call console without a subject
-```
+cy.console() // error about how you can't call console without a subject
+``
 
 {% note info %}
```

Comment:
@lilaconlee Not sure what happened here. It removed the backtick so now the codeblock wrapping is messed up https://github.com/cypress-io/cypress-documentation/blob/7b7c9cdeabf0f0c44a881606a78884bbc68329fa/source/api/cypress-api/custom-commands.md